src/Components/App/App.jsx

```diff
@@ -0,0 +1,37 @@
+import "./App.css";
+import { BrowserRouter as Router, Route, Routes } from "react-router-dom";
+
+// Component Imports
+import NavBar from "../Navbar/Navbar";
+import About from "../About/About";
+import Projects from "../Projects/Projects";
+import Footer from "../Footer/Footer";
+
+// MUI Imports
+import { Grid } from "@mui/material";
```

Comment:
Real nice organization of your imports here.  Keeping this clean and organized is super nice and honestly something my team could work on.